fighters/common/src/function_hooks/get_param.rs

```diff
@@ -77,6 +77,12 @@ pub unsafe fn get_param_float_hook(x0 /*boma*/: u64, x1 /*param_type*/: u64, x2
     // For articles
     let owner_module_accessor = &mut *sv_battle_object::module_accessor((WorkModule::get_int(boma, *WEAPON_INSTANCE_WORK_ID_INT_LINK_OWNER)) as u32);
 
+    // Coupled with "landing_heavy" change in change_motion hook
```

Comment:
This should be handled in params instead

Reply:
I didn't want to change this in params, as it would be confusing to someone without context why heavy landing lag was 1 frame less ingame than defined in params. This way the change can be easily altered/reverted if need be, as well.

Reply:
I see, my "param hook bad" mentality kicked in real quick on that one
In that case sure I'll allow it